addons/bus/models/bus_presence.py

```diff
@@ -65,11 +65,19 @@ def _update_presence(self, inactivity_period, identity_field, identity_value):
         if not presence:  # create a new presence for the user
             values[identity_field] = identity_value
             values['last_presence'] = last_presence
+            values["status"] = "online"
```

Comment:
We could receive an "away" presence from an idle browser on responding to a close.